src/web/app/components/session-edit-form/session-edit-form.component.ts

```diff
@@ -225,7 +225,18 @@ export class SessionEditFormComponent implements OnInit {
    * Handles session 'Help' link click event.
    */
   sessionHelpHandler(): void {
-    window.open(`${environment.frontendUrl}/web/instructor/help`);
-    // TODO scroll down to the session setup specific section in the help page
+    const helpWindow: Window | null = window.open(`${environment.frontendUrl}/web/instructor/help`);
```

Comment:
It is possible to append something like `#sessions-link` at the end of the URL instead of using the lengthly code?

https://stackoverflow.com/a/484781

Reply:
It seems not to work with angular, because angular drops things after hash sign (#) in the url.
Related issue: https://github.com/angular/angular/issues/6595

Reply:
The issue is closed (fixed) :P

https://github.com/angular/angular/issues/6595#issuecomment-230096985

Reply:
Maybe this would work. https://github.com/angular/angular/issues/6595#issuecomment-244232725 which will de-couple the scrolling logic from the page that uses the help page.

Reply:
I think the current one is just similar to this solution https://github.com/angular/angular/issues/6595#issuecomment-244232725
First get the element in the page and call `scrollIntoView()` method